### DISCUSSION TITLE
Welcome to react-adobe-animate Discussions!

### DISCUSSION THREAD

**Discussion by bibixx** [bibixx/react-adobe-animate#45] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by thangaraj80** [comment]
Hi Bibix,
How to control the movie clip created in adobe animate cc without ActionScript, in need to control using reactjs component?
I like to drag and drop and hide movie clips using react.js.

**Comment by mrehman29** [comment]
After adding react-adobe-animate to project animation is working file but all test start failing with error 

Test suite failed to run

    AnimateCC: AdobeAn dependency not found

can you suggest on fixing this

**Reply by bibixx** [reply]
🤔 Using this package in tests could be really tricky at this point as I am checking for the presence of the `AdobeAn` and `CreateJS` dependencies. I think I can come up with a solution that would be compatible with the test but that would require some changes in the package's source code. Hopefully I'll have some time in the nearest future to take a look at this. For now the best solution would probably be to mock the whole `react-adobe-animate` package for tests
